### PR TITLE
Update Makefile

### PR DESCRIPTION
**Pull request by vrde**
Solution: we already have a nice Makefile, but it's outdated. The idea is to revamp it and make it easy to use.

The description can be found in the updated `README.md` file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,30 @@
...
+```
+git clone https://github.com/bigchaindb/bigchaindb.git
+cd bigchaindb
+make run
+```
+
+To know the IP of the instance running, execute in a new terminal `make ip`.
+
+There are also other commands you can execute:
+- `make start`: Run BigchainDB from source and damonize it (stop it with `make stop`).
```
Comment (codegeschrei) [inline README.md]: daemonize ?

Comment (vrde) [reply README.md]: Weirdly enough, [yes](https://en.wikipedia.org/wiki/Daemon_(computing)).

Comment (codegeschrei) [reply README.md]: Ah, can you add an 'e' then? :D

Comment (vrde) [reply README.md]: damonizee?

Comment (codegeschrei) [reply README.md]: we can replace it with an evil emoji!

**Pull request merged**